overlord/snapstate/backend/fontconfig.go

```diff
@@ -31,7 +31,7 @@ var commandFromSystemSnap = cmdutil.CommandFromSystemSnap
 
 // updateFontconfigCaches always update the fontconfig caches
 func updateFontconfigCachesImpl() error {
-	for _, fc := range []string{"fc-cache-v6", "fc-cache-v7"} {
+	for _, fc := range []string{"fc-cache-v6", "fc-cache-v7", "fc-cache-v7.5"} {
```

Comment:
IIRC fc-cache-v7 and fc-cache-v7.5 both generate UUID based hashes (needs double checking though). It's probably enough to build and run the latter.

Reply:
I pushed up a change doing this, let's see how far it gets us

Reply:
seems to work, thanks for the suggestion!